src/conversions/cart.cpp

```diff
@@ -121,8 +121,15 @@ static double geocentric_radius(double a, double b, double cosphi,
         This is from WP2, but uses hypot() for potentially better
         numerical robustness
     ***********************************************************************/
+#ifdef not_optimized
     return hypot(a * a * cosphi, b * b * sinphi) /
```

Comment:
This does not compile even when `not_optimized` is defined. Should it be a comment instead?

Reply:
good point. commit ammended